### DISCUSSION TITLE
Icono de app anímate for android

### DISCUSSION THREAD

**Discussion by josquinet** [airsdk/Adobe-Runtime-Support#2449] [General]
Buenos días. cuando animate me pide en air configuration para android los logos, el tamaño mas grande que me pide es 192x192 px. Sin embargo, creo que el tamaño óptimo debería ser 512x512 px. ¿Por qué pienso esto? porque, justo antes de lanzar la aplicación en mi teléfono, el icono aparece pequeño y no ocupa todo el espacio que ofrece android, sino que queda en el centro y hay espacios en blanco a los lados (quizás el problema sea otro).... ¿hay alguna forma de subir un logo en 512?. ¿O solucionar el problema de los lados en blanco en mi icono justo antes de iniciar la aplicación? Muchísimas gracias

**Comment by teotigraphix** [comment]
Check this out:

https://developer.android.com/distribute/google-play/resources/icon-design-specifications

Probably what is going on, your icons are not updated to the new layouts.

**Reply by josquinet** [reply]
hello. It's actually what I thought, the problem is that in animate, in air settings for android, it won't let me upload an icon in 512x512. only up to 192x192....
How can I upload it in 512 by 512? play store forces this measure... (I send image)
thank you
![icono problema](https://user-images.githubusercontent.com/85651150/215963212-a0610040-063e-4ca2-8201-6c135cd55991.jpg)

**Comment by shaucky** [comment]
The tool in Adobe Animate is not very convenient for developer. It's just convenient for beginner.

Actually, what you input to the tool, would be write in a descriptor file, which named like "**your FLA file name-app.xml**". You can open the XML file, and write nodes for icons with other size.

The name of nodes are specified as this reference: https://help.adobe.com/zh_CN/air/build/WSfffb011ac560372f2fea1812938a6e463-8000.html#WSfffb011ac560372f-6fd06f0f1293d3b33ea-7ffc (I'm not sure your language, you can change the language code in the link to reach your language version).

By the way, i've read the document about descriptor file in https://airsdk.dev/ , it didn't list size of icons.@ajwfrost  And i have a doubt that could you also put referance in other language on the site? The English referance is hard to understand for me so i still always go to the  referance that hosted by Adobe because it has my language ver.

**Reply by ajwfrost** [reply]
@xiaoChi888  the translations are a bit of a problem for us, e.g. for the AS3 documentation we didn't get the localised versions from Adobe so looking to translate all the technical documentation is quite a large effort. I had hoped that online translation tools available via the main browsers might be sufficient for things instead though? Do you see that "translate this page" kind of option on the right hand end of the browser's URL bar?

**Reply by shaucky** [reply]
I thought i've seen it and it didn't matter but no. I verified just now, in Microsoft Edge, translate option didn't show on the end of URL bar when i browse https://airsdk.dev/reference/actionscript/3.0/ . And if i select some text and right click to select translate item, although the translate option appeared, but no matter how long it takes, it didn't translate anything… So my conclusion is… the translate tool didn't work on the referance page…
Not to mention Google Chrome, Google Translate exit mainland China in recent months…

**Reply by josquinet** [reply]
Muchísimas gracias por la respuesta xiaochi888. entiendo que debo cambiar algo en el archivo  **su nombre de archivo FLA-app.xml **. Este cambio lo hago para que luego en animate la configuracion de air for android me deje incluír un logo de 512?. 
querría añadir que tengo dos archivos xml, el que hice referencia antess y otro que se llama igual pero con la palabra RECUP_ al principio. este archivo incluye las siguientes líneas:

`<image36x36>iconos/iconos%20siglas/icono%2036.png</image36x36>`

`<image48x48>iconos/iconos%20siglas/icono%2048.png</image48x48>`

`<image72x72>iconos/iconos%20siglas/icono%2072.png</image72x72>`

`<image96x96>iconos/iconos%20siglas/icono%2096.png</image96x96>`

`<image144x144>iconos/iconos%20siglas/icono%20144.png</image144x144>`

`<image192x192>iconos/iconos%20siglas/icono%20192.png</image192x192>`

yo he añadido esta línea y no ha funcionado:

`<image512x512>iconos/iconos%20siglas/icono%20192.png</image512x512>`

quizás hay que cambiar el número 20192....estas son las investigaciones que he echo. también en el archivo  **su nombre de archivo FLA-app.xml ** he puesto esta línea:
`-<icon>`

`<image512x512>logo%20cascos/logo_cascos_grande_azul_512.png</image512x512>`

`</icon>`

 con un archivo en el destino señalado pero no me ha funcionado tampoco.

agradezco muchísimo la ayuda.

**Reply by shaucky** [reply]
I'm not sure what's causing your problem. Just the other day, when i set a FLA file from "AIR for Desktop" to "AIR for Android", it had a problem seems like the problem you have.

Even the target platform has changed, they should've use the same descriptor file, But icons were not referenced correctly when the application packaging, and the tool in Adobe Animate didn't show any icon. 

What i've done are copying the XML nodes, saving the descriptor file by the tool in Adobe Animate (it overwrite the icon nodes by blank space ), then pasted the nodes to the descriptor file again, and open Adobe Animate again. Icons had referenced correctly finally.

Wishing those would help you.

**Reply by shaucky** [reply]
`%20` is space in URI encoding. So you shouldn't delete the "20" in it.

**Reply by josquinet** [reply]
Muchas gracias!
